lib/util.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
+const path = require('path');
 const { parse } = require('url');
 const querystring = require('querystring');
 const pathabs = require('path-is-absolute');
```

Comment:
This should be gone by now https://github.com/webpack/webpack-dev-middleware/pull/335. Could you rebase with the PR with latest `master` to ensure we won't get a bad merge here please ?

Reply:
Done 😄

Reply:
Doesn't seemed to be the case the line is still there (?)

Reply:
Ahh, sorry, forgot to `push -f` 😅
Should be fixed now.